pkg/controller/controllerutil/controllerutil.go

```diff
@@ -18,11 +18,13 @@ package controllerutil
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"os"
 	"reflect"
 
 	"k8s.io/apimachinery/pkg/api/equality"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Didn't know mixed-case import aliasing is allowed

Reply:
changed to lowercase anyway